src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1260,6 +1260,24 @@
   <data name="DoNotUseStackallocInLoopsMessage" xml:space="preserve">
     <value>Potential stack overflow. Move the stackalloc out of the loop.</value>
   </data>
+  <data name="PreferStreamReadAsyncMemoryOverloadsTitle" xml:space="preserve">
+    <value>Prefer Memory&lt;Byte&gt; overload for ReadAsync.</value>
```

Comment:
```suggestion
    <value>Prefer 'Memory&lt;Byte&gt;' overload for 'ReadAsync'.</value>
```

Reply:
Suggest similar changes for all resource strings - putting type and keyword names in quotes makes it more readable.